atomic-git/f22_spc/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# based on Dockerfile by Brent Baude
+
+FROM openscap-daemon-f22
+
+RUN dnf -y update && dnf -y groupinstall "Development Tools" && dnf -y install make wget git && dnf clean all
+
+RUN dnf remove -y openscap-containers && \
+    dnf -y install autoconf automake libtool curl-devel libxml2-devel \
```

Comment:
`dnf builddep openscap && dnf install libtool`

should do the trick. The point is that all the dependencies are in the openscap.spec in Fedora. And once they change, you will get them for free.